.gitignore

```diff
@@ -29,6 +29,7 @@ dlangspec.tex
 
 /web
 /2.*.ddoc
+modlist-*.ddoc
```

Comment:
```
@echo You should issue manually: rm -rf ${DMD_DIR}-${LATEST} ${DRUNTIME_DIR}-${LATEST} ${PHOBOS_DIR}-${LATEST} ${STABLE_DMD_ROOT} ${DUB_DIR}
```

No, ideally, telling the user to manually clean up your crud should not be necessary.